content/modules/python/pages/sets.adoc

```diff
@@ -0,0 +1,190 @@
+= Sets
+
+== Introduction
+
+In Python, lists and sets are both mutable. The key difference is that sets are _unordered_ and cannot have duplicate elements in a set. Sets are particularly useful for membership testing and eliminating said duplicates, even if lists are more useful overall.
```

Comment:
mutable= inconsistent, varying, shifting.... can you edit to a different word?

Reply:
Is that necessary? Mutable and immutable are both really common words when referring to the ability to change data structures.

Reply:
do we have this definition cited somewhere or can we refer to it as fixed or adjustable? you are correct that it cant be inconsistent or a variable or shifting but I do think that we either need to have this definition cited  so saying mutable=adjusting or immutable =fixed

Reply:
Yes, there are 4 instances within the Examples Book where mutable is used and defined as a term. The Python Variables page, the Python Tuples page, the Python Lists page, and the first question of the first project for STAT19000 students. Are you wanting me to link to a definition of mutable?

Reply:
that would be awesome!